Moderation-Policy.md

```diff
@@ -54,9 +54,9 @@ this default policy applies.
 * *CommComm* refers to the [Node.js Community Committee][].
 * *Post* refers to the content and titles of any issue, pull request, comment
   or wiki page.
-* *Moderate* refers to the act of modifying the content and title of, or
-  deleting, any Post for the purpose of correcting or addressing Code of Conduct
-  violations.
+* *Moderate* refers to the act of modifying the content and title of, locking or
+  deleting, any Post or thread for the purpose of correcting or addressing Code
```

Comment:
What's a thread in this context? (If a "thread" is made up of posts, I don't think we need to specify it.)

Reply:
A thread is made up of posts. it is a PR or an ISSUE

Reply:
I pushed a commit with my suggestion to improve this definition. Force-push it out if you hate it. Thanks.